tasks/vmaas_sync/package_sync.go

```diff
@@ -218,15 +235,62 @@ func storePackageDetailsFrmPkgListItems(tx *gorm.DB, nameIDs map[string]int, pkg
 	}
 	utils.Log("packages", len(toStore)).Info("Collected packages to store")
 
+	err := toStore.storeOrUpdate(tx)
+	return err
+}
+
+//nolint:funlen
+func (pkgs pkgSlice) storeOrUpdate(tx *gorm.DB) error {
+	var toUpdate pkgSlice
+
+	nameIDEvra := make([][]interface{}, 0, len(pkgs))
```

Comment:
If there is no reason for interface maybe let's use `nameIDandEvra` from 52 line